README.md

```diff
@@ -78,16 +67,76 @@ pnpm install
   cd src && cp config-examples/* config && for i in config/*-example.js; do mv -- "$i" "${i%-example.js}.js"; done
   ```
 
-4. Set the app-config.js values for:
+2. Set the app-config.js values for:
 
 - `orsApiKey` - ORS API key to be used when ot running the app from localhost or ors valid domains
 - `bitlyApiKey` - the Bitly key (used to shorten the share URL)
 - `bitlyLogin` - the Bitly login (used to shorten the share URL)
 
-5. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
+3. The ORS menu is loaded/used by default. If you want to use a custom menu, have a look in the `hooks-example.js`.
 
 The filters, theme and hooks of the map client can be customized if needed.
 
+##### Run with `Docker`
+
+Run the app in `docker` in 3 steps:
+
+1. Build the image:
+
+```shell
+docker build --tag ors-map-client:local .
+```
+
+2. Run a container:
+
+```shell
+docker run -d -it -v "$(pwd)"/nginx/logs:/var/log/nginx:rw -p 127.0.0.1:8080:80 --name ors-map-client ors-map-client:local
+```
+
+3. Go to http://127.0.0.1:8080 and enjoy!
+
+If you want to check the logs, run:
+
+```shell
+docker logs --follow ors-map-client
+```
+
+##### Run with `docker-compose`
+
+Run the app in `docker-compose` in 2 steps:
+
+1. Build and run the image:
+
+```shell
+docker-compose up -d
+```
+
+2. Go to http://127.0.0.1:8080 and enjoy!
+
+If you want to check the logs, run:
+
+```shell
+docker-compose logs -ft
```

Comment:
should we use the `docker compose` instead or is `docker-compose` (standalone thing) still the default?
I think compose comes with all Docker Desktop versions available as `docker compose`